From a9bf8b5d83221ae18cc737b4600d9b9a74359128 Mon Sep 17 00:00:00 2001
From: Chris Burroughs <chris.burroughs@joyent.com>
Date: Mon, 10 Oct 2016 16:54:32 -0400
Subject: [PATCH] RELENG-612 prefer (git+)https urls Reviewed by: Cody Peter
 Mello <cody.mello@joyent.com> Reviewed by: Trent Mick <trent.mick@joyent.com>
 Approved by: Trent Mick <trent.mick@joyent.com>

---
 .gitmodules  | 10 +++++-----
 package.json |  4 ++--
 2 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/.gitmodules b/.gitmodules
index 4f2fc71..68b60c6 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -1,15 +1,15 @@
 [submodule "deps/javascriptlint"]
 	path = deps/javascriptlint
-	url = git://github.com/davepacheco/javascriptlint.git
+	url = https://github.com/davepacheco/javascriptlint.git
 [submodule "deps/jsstyle"]
 	path = deps/jsstyle
-	url = git://github.com/davepacheco/jsstyle.git
+	url = https://github.com/davepacheco/jsstyle.git
 [submodule "deps/sdc-scripts"]
 	path = deps/sdc-scripts
-	url = git@github.com:joyent/sdc-scripts.git
+	url = https://github.com/joyent/sdc-scripts.git
 [submodule "deps/restdown"]
 	path = deps/restdown
-	url = git://github.com/trentm/restdown.git
+	url = https://github.com/trentm/restdown.git
 [submodule "deps/restdown-brand-remora"]
 	path = deps/restdown-brand-remora
-	url = git@github.com:joyent/restdown-brand-remora.git
+	url = https://github.com/joyent/restdown-brand-remora.git
diff --git a/package.json b/package.json
index c075736..19cc89b 100644
--- a/package.json
+++ b/package.json
@@ -15,9 +15,9 @@
         "lomstream": "^1.1.0",
         "lru-cache": "2.5.0",
         "mkdirp": "0.5.0",
-        "moray": "git+ssh://git@github.com:joyent/node-moray.git#b84ef0e",
+        "moray": "git+https://github.com/joyent/node-moray.git#b84ef0e",
         "node-uuid": "1.4.2",
-        "portolan-moray": "git+ssh://git@github.com:joyent/sdc-portolan-moray.git#72eb031",
+        "portolan-moray": "git+https://github.com/joyent/sdc-portolan-moray.git#72eb031",
         "vasync": "1.6.2",
         "verror": "1.6.0",
         "vstream": "0.1.0"
-- 
2.21.0

